docs/load-icons.js

```diff
@@ -567,6 +567,23 @@ module.exports = [
 		),
 	},
 
+	{
+		name: 'GetMaximumIcon',
+		component: getDefaultExport(
+			require('../src/components/Icon/GetMaximumIcon/GetMaximumIcon')
+		),
+		examplesContext: require.context(
+			'../src/components/Icon/GetMaximumIcon/examples',
+			true,
+			/\.jsx?$/
```

Comment:
No need to do it for this PR (it's not super important for such simple components) but new examples should we written in TS moving forward.

Reply:
Sounds good. Will do.